misc/website/README

```diff
@@ -0,0 +1,36 @@
+The script build_website.py builds the website locally, combining the following components:
+    - docs/ directory of the Fast Downward repository (= this repository)
+    - documentation of the search plugins generated by
+      misc/autodoc/generate-docs.py
+    - documentation from a clone of https://github.com/aibasel/downward-markdown
+
+The content is generated from the current local working copies of the repositories.
+In contrast to the public website, the generated website only contains the
+pages for this version (not for previous releases).
+
+Setup:
+------
+
+Create a python virtual environment and activate it:
+$ python -m venv .env
+$ source .env/bin/activate
+
+Install the requirements:
+$ python3 -m pip install -r requirements.txt
+
+Set the environment variable DOWNWARD_MARKDOWN to a local copy of the downward-markdown repository (https://github.com/aibasel/downward-markdown):
```

Comment:
Ah, breaking this line to stick to the fixed character width was fine. I meant removing the empty line between instruction and corresponding shell command. Same for the other line I commented on.